torch/__init__.py

```diff
@@ -36,20 +36,36 @@
 ################################################################################
 
 if platform.system() == 'Windows':
-    NVTOOLSEXT_PATH = os.getenv('NVTOOLSEXT_PATH', 'C:\\Program Files\\NVIDIA Corporation\\NvToolsExt')
-
-    if os.path.exists(NVTOOLSEXT_PATH):
-        nvtoolsext_lib_path = os.path.join(NVTOOLSEXT_PATH, 'bin', 'x64')
-    else:
-        nvtoolsext_lib_path = ''
-
     py_dll_path = os.path.join(sys.exec_prefix, 'Library', 'bin')
     th_dll_path = os.path.join(os.path.dirname(__file__), 'lib')
 
-    dll_paths = [th_dll_path, py_dll_path, nvtoolsext_lib_path, os.environ['PATH']]
+    if not os.path.exists(os.path.join(th_dll_path, 'nvToolsExt64_1.dll')):
+        nvtoolsext_dll_path = os.path.join(
+            os.getenv('NVTOOLSEXT_PATH', 'C:\\Program Files\\NVIDIA Corporation\\NvToolsExt'), 'bin', 'x64')
+    else:
+        nvtoolsext_dll_path = None
+    
+    import torch.version
```

Comment:
I know this may not no so good because it makes caffe2 depends on torch. But I don't know how to make it better.